django_scheduled_tasks/models.py

```diff
@@ -0,0 +1,43 @@
+import importlib
+from django.utils import timezone
+from django.db import models
+from django.core.validators import MinValueValidator, MaxValueValidator
+
+
+class ScheduledTask(models.Model):
+    """
+    A record represents a single scheduled (recurring) task
+    """
+    func = models.CharField(max_length=100, unique=True)  # importable function name
+    interval_minutes = models.IntegerField(validators=[MinValueValidator(1),
+                                                       MaxValueValidator(1440)])  # 1 min to 24 hours
+    enabled = models.BooleanField(default=True)
+    last_timestamp = models.DateTimeField(null=True, blank=True)
+    last_success = models.BooleanField(null=True, blank=True)
+
+    def execute(self):
+        """ A callable function, found using our string func value """
+        modulename, funcname = self.func.rsplit('.', 1)
```

Comment:
again minor, should words in variables be separated with _ ... e.g. modulename -> module_name for consistency with others that already are e.g. imported_module

Reply:
Fixed in 25815bd